eslintrc.base.js

```diff
@@ -92,5 +92,11 @@ module.exports = {
                 'security/detect-eval-with-expression': 'off',
             },
         },
+        {
+            files: ['src/background/**/*', 'src/injected/**/*'],
+            rules: {
+                '@typescript-eslint/no-floating-promises': ['warn'],
```

Comment:
I think we should probably leave this `off` rather than `warn` until we address the other issues; I don't think it makes sense to pollute unrelated PRs with warnings that aren't relevant to them